jenkins/README.md

```diff
@@ -72,6 +72,16 @@ See #<issue number>
 '
 gh pr create
 ```
+
+## Network Resources
+
+Downloading files from the Internet in CI is a big source of flaky failures
+(e.g. remote server goes down or is slow), so try to avoid using the network at
+all during tests. In some cases this isn't a reasonable proposition (e.g. the
+docs tutorials which need to download models). In these cases you can re-host
+files in S3 for fast access in CI. A committer can upload a file, specified by
+a name, hash, and path in S3, using the `workflow_dispatch` event on
+[the `upload_ci_resource.yml` GitHub Actions workflow](https://github.com/driazati/tvm/actions/workflows/upload_ci_resource.yml).
```

Comment:
can you fix private repo mention? also, what should folks use as upload_path?

Reply:
fixed, and `upload_path` is wherever the file goes in s3 so it can be whatever the person triggering the build wants (e.g. for the oneflow dep in #11839 the upload path is `oneflow/resnet18.zip`)